startup/index.js

```diff
@@ -0,0 +1,32 @@
+import apiClient from "@adaptable/client";
+
+const appId = process.env.ADAPTABLE_APP_ID;
+if (appId == null) throw new Error("No ADAPTABLE_APP_ID found");
+
+const revId = process.env.ADAPTABLE_APPREVISION_ID;
+if (revId == null) throw new Error("No ADAPTABLE_APPREVISION_ID");
+
+const imageBuildProps = {
+    appId,
+    config: {
+        type: "buildpack",
+        builder: "paketobuildpacks/builder:base",
+    },
+    imageName: "appimage",
+    plan: "hobby",
+    revId,
+};
+
+// This will initiate a build with the same parameters that adapt will use
+// When adapt sees the build already in progress, it will adopt the build
+// as its own.
+// However, the build will have started in parallel with the adapt startup
+// reducing the total time to app ready.
+async function doBuild(props) {
+    const client = await apiClient.default();
+    await client.service("builds").create(imageBuildProps);
+}
+
+doBuild(imageBuildProps)
+    .then(() => {})
```

Comment:
I think both your then and catch blocks are extraneous as currently written. Definitely delete the then, but you should probably handle the error in  your top level catch by printing the error and exiting with a non-zero code. I don't think you want to rely on the default unhandled promise rejection behavior.

Reply:
I'd like to keep the top-level catch so that we don't get an unhandled promise rejection in any circumstance.  See if you are ok with what is there now.